### PR TITLE
[ROX-10916]: Add a new security policy reference topic

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): RHACS 3.72.0

Issue: https://issues.redhat.com/browse/ROX-10916

Link to docs preview: https://openshift-docs-git-security-policy-ref-lahinson.vercel.app/openshift-acs/master/operating/default-security-policies.html

Additional information:

### CODE REVIEW

**Comment by lahinson** [comment]
@lmaynard88 When you can, PTAL at my first draft of the security policies reference topic. The preview link is in the first comment 👆 

Per Boaz's feedback, I created multiple tables, sorted by severity. Within each table, I sorted the policies by life cycle stage. I also sent Boaz a preview link on Slack.

**Comment by lmaynard88** [comment]
Thanks, @lahinson! A couple of comments: 
- Perhaps a separate page for this reference content with a TOC entry after Managing Security Policies might make sense? 
- I see the severity mentioned in the table titles. This might be easy to miss. Could a column be added or perhaps the severity could be added to the enabled/disabled column? 
- Another option could be to use letters in a severity column (C, H, M, L) and also in the enabled/disabled (E, D) one.

**Comment by lahinson** [comment]
@lmaynard88 I updated the topic per your first suggestion, and I think the changes help to more clearly call out the severity levels. I didn't abbreviate any terms yet because the content seems to be fitting well in the tables so far. Let me know what you think when you can. Thanks! 

Preview link (VPN required): http://file.rdu.redhat.com/lahinson/security-policy-ref/operating/default-security-policies.html

**Comment by lmaynard88** [comment]
Thanks, @lahinson. I agree the severity levels are more obvious with the changes. I like it on a separate page as well.

**Comment by kcarmichael08** [comment]
I like the way this is organized! It looks good, and although our options for sorting are limited, there's always ctrl-F. :) There are a couple of policies that are new that (I think) aren't listed - probably because they got included before the list was generated for you to document. There's "Container with privilege escalation allowed" https://docs.openshift.com/acs/3.71/release_notes/371-release-notes.html#privilege-escalation which is Deploy stage, enabled by default, and Medium severity. There's "Deployments with externally exposed endpoints" - https://docs.openshift.com/acs/3.71/release_notes/371-release-notes.html#externally-exposed-service - which is Deploy stage, disabled by default, and Medium severity.

**Comment by lahinson** [comment]
Thanks for the review, @kcarmichael08! I added the two policies that you mentioned. I'm having some issues with creating a local preview this afternoon, but I hope to have an updated preview soon.

File: modules/low-sev-security-policies.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+[options="header"]
+|====
+|Life cycle stage |Name |Description | Status
+|Build or Deploy |90-Day Image Age |Alerts when a deployment has not been updated in 90 days. |Enabled
+|Build or Deploy |ADD Command used instead of COPY |Alerts when a deployment uses an ADD command. |Enabled
+|Build or Deploy |Alpine Linux Package Manager (apk) in Image |Alerts when a deployment includes the Alpine Linux package manager (apk). |Enabled
+|Build or Deploy |Curl in Image |Alerts when a deployment includes curl. |Disabled
+|Build or Deploy |Docker CIS 4.1: Ensure That a User for the Container Has Been Created |Ensures that containers are running as non-root users. |Enabled
+|Build or Deploy |Docker CIS 4.7: Alert on Update Instruction |Ensures that update instructions are not used alone in the Dockerfile. |Enabled
+|Build or Deploy |Insecure specified in CMD |Alerts when a deployments uses 'insecure' in the command. |Enabled
```
Comment (jldohmann) [inline modules/low-sev-security-policies.adoc:27]:
```suggestion
|Build or Deploy |Insecure specified in CMD |Alerts when a deployment uses 'insecure' in the command. |Enabled
```

File: modules/medium-sev-security-policies.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+|Build |Docker CIS 4.4: Ensure images are scanned and rebuilt to include security patches |Alerts when images are not scanned and rebuilt to include security patches. It is important to scan images often to find vulnerabilities, rebuild the images to include security patches, and then instantiate containers for the images. |Disabled
+|Deploy |30-Day Scan Age |Alerts when a deployment has not been scanned in 30 days. |Enabled
+|Deploy |CAP_SYS_ADMIN capability added |Alerts when a deployment includes containers that are escalating with CAP_SYS_ADMIN. |Enabled
+|Deploy |Container using read-write root filesystem |Alerts when a deployment includes containers that have read-write root file systems. |Enabled
+|Deploy |Container with privilege escalation allowed |Alerts when a container might be running with unintended privileges, creating a security risk. This situation can happen when a container process that has more privileges than its parent process allows the container to run with unintended privileges. |Enabled
+|Deploy |Deployments should have at least one ingress Network Policy |Alerts if deployments are missing an ingress Network policy. |Enabled
```
Comment (jldohmann) [inline modules/medium-sev-security-policies.adoc:27]: Should "Network" be capitalized here? [Ingress](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#ingress) is usually capitalized

**Review by jldohmann — COMMENTED** [review]
Looking good! One minor nit and a question

File: modules/medium-sev-security-policies.adoc (hunk 2 above)
Comment (lahinson) [reply modules/medium-sev-security-policies.adoc:27]: Good question! Per the Kubernetes glossary, "Network Policy" is capitalized. I'll update the topic to capitalize "ingress" and the second instance of "policy"

**Review by jldohmann — APPROVED** [review]
/lgtm

**Pull request merged**